src/Blocks/BlockChest.h

```diff
@@ -84,7 +84,12 @@ class cBlockChestHandler :
 
 	virtual bool CanBeAt(cChunkInterface & a_ChunkInterface, const Vector3i a_RelPos, const cChunk & a_Chunk) override
 	{
-		auto BlockPos = a_Chunk.RelativeToAbsolute(a_RelPos);
+		if (!cChunkDef::IsValidHeight(a_RelPos.y - 1))
+		{
+			return false;
+		}
```

Comment:
This doesn't belong here, we aren't reading any block at `a_RelPos.y - 1`.

Reply:
Fixed